variables.tf

```diff
@@ -42,6 +42,18 @@ variable "aws_account_password_policy" {
   description = "AWS account password policy parameters for the audit, logging and master account"
 }
 
+variable "aws_auditmanager_config" {
+  type = object({
+    enabled             = bool
+    reports_bucket_name = string
+  })
+  default = {
+    enabled             = true
+    reports_bucket_name = "audit-manager-reports"
```

Comment:
can you change reports_bucket_name to reports_bucket_prefix

Reply:
Will update